pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/SegmentStatusChecker.java

```diff
@@ -291,6 +295,7 @@ void setStatusToDefault() {
       _metricsRegistry.setValueOfTableGauge(tableName, ControllerGauge.NUMBER_OF_REPLICAS, 0);
       _metricsRegistry.setValueOfTableGauge(tableName, ControllerGauge.PERCENT_OF_REPLICAS, 0);
       _metricsRegistry.setValueOfTableGauge(tableName, ControllerGauge.SEGMENTS_IN_ERROR_STATE, 0);
+      _metricsRegistry.setValueOfTableGauge(tableName, ControllerGauge.PERCENT_OF_SEGMENTS, 0);
```

Comment:
Please add some comments here on what these metrics mean. is this percent of segmens that are Avaialble, or those that are Unavailable? In fact, the PERCENT_OF_REPLICAS should also have been named correctly. PERCENT_REPLICAS_UNAVAILABLE. In this case, it is percent of segments that are unavailable, right?